src/ServiceControlInstaller.Engine/Configuration/ServiceControl/SettingsList.cs

```diff
@@ -45,7 +45,8 @@ public static class ServiceControlSettings
         public static SettingInfo AuditRetentionPeriod = new SettingInfo
         {
             Name = "ServiceControl/AuditRetentionPeriod",
-            SupportedFrom = new Version(1, 12, 1)
+            SupportedFrom = new Version(1, 12, 1),
+            RemovedFrom = new Version(4, 0, 0)
```

Comment:
We need to drop this. We can't deprecate the setting until we remove audit data from the main instance

Reply:
Good catch forgot to remove